keyboards/ciaanh/kanagawa/config.h

```diff
@@ -0,0 +1,87 @@
+/* Copyright 2023 Ciaanh (@ciaanh)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+/* OLED SPI Defines */
+#ifdef OLED_ENABLE
+#    define OLED_DISPLAY_128X32
+#    define OLED_FONT_H "lib/oledfont.c"
+#    define OLED_FONT_END 255
```

Comment:
Your font file isn't using the last row, so the default of 225 for the oled font size should be more than sufficient. 

```suggestion
```